docs/howto/manage-web-app-charms/use-web-app-charm.rst

```diff
@@ -29,6 +29,230 @@ can be run multiple times) and that it can be run on multiple units simultaneous
 without issue. Handling multiple migration scripts that run concurrently
 can be achieved by, for example, locking any tables during the migration.
 
+Troubleshoot the charm
+----------------------
+
+This guide provides helpful commands to run in situations where your web app
+deployment is not successful (for instance, if your app crashes).
+
+Get the app logs
+~~~~~~~~~~~~~~~~
+
+The first step of troubleshooting is to review the logs for errors. They are
```

Comment:
Perhaps just frame this as one way rather than the first - it depends on individual preference